django/db/backends/sqlite3/creation.py

```diff
@@ -4,11 +4,15 @@
 
 from django.core.exceptions import ImproperlyConfigured
 from django.db.backends.base.creation import BaseDatabaseCreation
+from django.utils.encoding import force_text
 from django.utils.six.moves import input
 
 
 class DatabaseCreation(BaseDatabaseCreation):
 
+    def is_database_in_memory(self, database_name):
```

Comment:
> Do you intentionally name the methods differently?

I did that so it's more obvious when looking at the code which version is being called (the connection version or this version). But I'm not wedded to a different name.

> Should we make it a @staticmethod since self isn't used?

I kept it a method as opposed to making a function to preserve the possibility of overriding with different implementation (though I'm not sure if subclassing DatabaseCreation is actually supported at the moment). As for `@staticmethod`, that did occur to me. I'm not well-versed in the nuances, but I was under the impression that it doesn't behave as nicely with respect to inheritance, etc, so I thought it best not to get too fancy there. Class method is probably better though. What do you think?